algosdk/error.py

```diff
@@ -222,3 +222,8 @@ def __init__(self, msg):
 class AtomicTransactionComposerError(Exception):
     def __init__(self, msg):
         super().__init__(msg)
+
+
+class InvalidForeignAppIdError(Exception):
```

Comment:
are you planning to also have `InvalidForeignAssetError` and similarly for **app accounts** and **boxes** ?

Reply:
I wasn't planning to, but maybe it would be better to generalize this into something like: `InvalidForeignIndexError`?